build_tools/services.rb

```diff
@@ -10,10 +10,10 @@ class ServiceEnumerator
     MANIFEST_PATH = File.expand_path('../../services.json', __FILE__)
 
     # Minimum `aws-sdk-core` version for new gem builds
-    MINIMUM_CORE_VERSION = "3.197.0"
+    MINIMUM_CORE_VERSION = "3.197.2"
```

Comment:
I actually don't think we need to bump this.  This is just a minor fix in core.  All service gems will continue to work (ie, this does not change service gems in any way that requires the new core version to work).

Reply:
I am worried about old service, new core, and new service, old core. I've reverted this.